sentry_streams/sentry_streams/deployment_config/transformer_config.yaml

```diff
@@ -0,0 +1,37 @@
+env:
+  topics:
+    events: events
+    transformed-events: transformed-events
+    transformed-events-2: transformed-events-2
+
+pipeline:
+  segments:
+    my_segment:
+      parallelism: 1 # This is runtime-agnostic
+      steps_config:
+        myinput:
+          starts_segment: True
+          common:
+            type: source
+            bootstrap_servers: localhost:9092
+            consumer_group: python-test
+            auto_offset_reset: latest
+          flink_overrides:
+
+        parser:
+          common:
+            type: map
```

Comment:
These `type` fields are only needed for `source` and `sink` steps. This is because the way the Arroyo adapter works, where we set up the Kafka consumer and producer configuration at the `build` step, instead of doing this when we are processing a Source or Sink Step.